src/client/e2e/credit.spec.ts

```diff
@@ -92,4 +91,44 @@ test.describe("Credit", () => {
       expect(tradeId).toEqual(blotterId)
     })
   })
+
+  test.describe("Sell side", () => {
+    test.only("Sell side ticket", async ({ context }) => {
+      await newRfqPage.getByPlaceholder(/Enter a CUSIP/).click()
+      await newRfqPage
+        .locator("[data-testid='search-result-item']")
+        .nth(5)
+        .click()
+
+      const quantity = newRfqPage.locator("[data-testid='quantity']")
+      await quantity.type("2")
+      await quantity.blur()
```

Comment:
minor: wot's the `.blur()` for?

Reply:
side note .. cursor management is ****ed in the Dealer checkbox list ...

Reply:
copy/paste artifact, will remove

Reply:
not sure what you mean by the dealer checkbox list

Reply:
in the UI .. the list of Dealers that you choose (Adaptive Bank etc.) .. hovering over and moving down the list is flaky (not a now thing, dw)